Code/Framework/AzCore/AzCore/IO/SystemFile.cpp

```diff
@@ -97,9 +96,6 @@ SystemFile& SystemFile::operator=(SystemFile&& other)
 
 bool SystemFile::Open(const char* fileName, int mode, int platformFlags)
 {
-    AZ_PROFILE_INTERVAL_SCOPED(AZ::Debug::ProfileCategory::AzCore, this, "SystemFile::Open - %s", fileName);
-    AZ_PROFILE_SCOPE_STALL_DYNAMIC(AZ::Debug::ProfileCategory::AzCore, "SystemFile::Open - %s", fileName);
```

Comment:
Especially the `AZ_PROFILE_SCOPE_STALL_DYNAMIC` is really helpful in finding file that mess with streaming. With the native file streaming (at least on Windows) and Asset Streaming, SystemFile shouldn't be used during level, only for early boot and some nice uses. If these are called during loading they interfere with file streaming so these markers are really helpful in finding those cases and in particular which files aren't playing nice. It would be appreciated if they can come back in some form.

Reply:
The events that aren't part of the set of the "lowest common denominator" of functionality that all profilers support will be refactored into a separate header in the future.

Reply:
I'm more than happy to have these not be part of the general default set. It would actually be great to have a CVar to allow these to be turned on/off. Deleting them now seems a bit premature though because they'll be gone rather than left for a future update.